Mathlib/Data/ZMod/Quotient.lean

```diff
@@ -0,0 +1,210 @@
+/-
+Copyright (c) 2021 Anne Baanen. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anne Baanen
+
+! This file was ported from Lean 3 source module data.zmod.quotient
+! leanprover-community/mathlib commit da420a8c6dd5bdfb85c4ced85c34388f633bc6ff
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Data.ZMod.Basic
+import Mathlib.GroupTheory.GroupAction.Quotient
+import Mathlib.RingTheory.Int.Basic
+import Mathlib.RingTheory.Ideal.QuotientOperations
+import Mathlib.Algebra.Hom.Equiv.TypeTags
+
+/-!
+# `ZMod n` and quotient groups / rings
+
+This file relates `ZMod n` to the quotient group
+`ℤ / AddSubgroup.zmultiples (n : ℤ)` and to the quotient ring
+`ℤ ⧸ Ideal.span {(n : ℤ)}`.
+
+## Main definitions
+
+ - `ZMod.quotientZmultiplesNatEquivZmod` and `ZMod.quotientZmultiplesEquivZmod`:
+   `ZMod n` is the group quotient of `ℤ` by `n ℤ := AddSubgroup.zmultiples (n)`,
+   (where `n : ℕ` and `n : ℤ` respectively)
+ - `ZMod.quotient_span_nat_equiv_zmod` and `ZMod.quotientSpanEquivZmod `:
+   `ZMod n` is the ring quotient of `ℤ` by `n ℤ : Ideal.span {n}`
+   (where `n : ℕ` and `n : ℤ` respectively)
+ - `ZMod.lift n f` is the map from `ZMod n` induced by `f : ℤ →+ A` that maps `n` to `0`.
+
+## Tags
+
+zmod, quotient group, quotient ring, ideal quotient
+-/
+
+
+open QuotientAddGroup
+
+open ZMod
+
+variable (n : ℕ) {A R : Type _} [AddGroup A] [Ring R]
+
+namespace Int
+
+/-- `ℤ` modulo multiples of `n : ℕ` is `ZMod n`. -/
+def quotientZmultiplesNatEquivZmod : ℤ ⧸ AddSubgroup.zmultiples (n : ℤ) ≃+ ZMod n :=
```

Comment:
Shouldn't this be `ZMod`? (Everywhere in this file.) I would also suggest `ZMultiples` but it seems `Zmultiples` is used consistently in other files.
```suggestion
def quotientZmultiplesNatEquivZMod : ℤ ⧸ AddSubgroup.zmultiples (n : ℤ) ≃+ ZMod n :=
```